src/amo/components/SearchForm/index.js

```diff
@@ -143,7 +144,12 @@ export class SearchFormBase extends React.Component {
       return;
     }
 
-    if (value.length > SEARCH_TERM_MAX_LENGTH) {
+    if (value.length < SEARCH_TERM_MIN_LENGTH) {
+      log.debug(oneLine`Search term length is too short to dispatch autocomplete action. Minimum length is: ${SEARCH_TERM_MIN_LENGTH}`);
+      // Reset the result set to prevent rendering a previously valid result set
+      this.props.suggestions.length = 0;
```

Comment:
Mutating the `props` object should be avoided (React props are immutable), so it would be best to handle this another way.